build_runner/lib/src/generate/directory_watcher_factory.dart

```diff
@@ -3,6 +3,8 @@
 // BSD-style license that can be found in the LICENSE file.
 import 'package:watcher/watcher.dart';
 
+export 'package:watcher/watcher.dart' show DirectoryWatcher;
```

Comment:
I don't think we should export things from other packages generally - if this api changed then that would now be a breaking change in `build`.